File: daily/867. 转置矩阵.js
/**
 * @param {number[][]} A
 * @return {number[][]}
 */
var transpose = function(A) {
    var B=[];
    for(var i=0;i<A[0].length;i++){
        var temp=[];
        for(var j=0;j<A.length;j++){
            temp.push(A[j][i]);
        }
        B.push(temp);
    }
    return B;
};

File: daily/867. 转置矩阵.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& A) {
        vector<vector<int>>B;
        B.resize(A[0].size());
        for(int i=0;i<B.size();i++)
            B[i].resize(A.size());
        
        for(int i=0;i<A.size();i++)
            for(int j=0;j<A[0].size();j++)
                B[j][i]=A[i][j];
        return B;
    }
};
